tox.ini

```diff
@@ -38,8 +38,14 @@ use_develop = False
 deps =
 	ipython
 commands =
+  python -m 'print("Simple discovery performance")'
```

Comment:
looks like you typoed these, should have been `python -c ...`  This breaks CI.

(PS: the reason I used Python for the prints is for OS-independence.)

Reply:
Gah. Yep, I messed up. Should have tested. Fixed in 61a265c.